wagtail/wagtaildocs/views/chooser.py

```diff
@@ -49,6 +49,7 @@ def chooser(request):
         documents = hook(documents, request)
 
     q = None
+    saved_documents = Document.objects.all()
```

Comment:
Since you never use the actual documents objects, I reckon this could use `Document.objects.exists()` instead. In this case, a variable name like `has_documents` will better express the intent.

In addition, this can probably live within the `if` statement so it only runs whenever the variable is actually used.